### PR TITLE
Document service defaults/admission plugins location

### PR DESCRIPTION
**Pull request by moelsayed**
https://github.com/rancher/rke/issues/1265

### CODE REVIEW

File: content/rke/latest/en/config-options/services/services-extras/_index.md [hunk 1]
```diff
@@ -9,10 +9,14 @@ RKE supports additional service arguments, volume binds and environment variable
 
 For any of the Kubernetes services, you can update the `extra_args` to change the existing defaults.
 
-As of `v0.1.3`, using `extra_args` will add new arguments and **override** any existing defaults. For example, if you need to modify the default admission controllers list, you need to include the default list and edit it with your changes so all changes are included.
+As of `v0.1.3`, using `extra_args` will add new arguments and **override** any existing defaults. For example, if you need to modify the default admission plugins list, you need to include the default list and edit it with your changes so all changes are included.
 
 Prior to `v0.1.3`, using `extra_args` would only add new arguments to the list and there was no ability to change the default list.
 
+> **Note:** All service defaults and paramters are defined per [`kubernetes_version`]({{< baseurl >}}/rke/latest/en/config-options/#kubernetes-version). A detailed list of the options can be found [here](https://github.com/rancher/kontainer-driver-metadata/blob/master/rke/k8s_service_options.go)
```
Comment (deniseschannon) [inline content/rke/latest/en/config-options/services/services-extras/_index.md]: can we distinguish between the versions? (RKE v0.3.0+) versus prior to RKE v0.3.0? The file locations are different between the two. One comes from kontainer-driver-metadata and the other comes from types.

File: content/rke/latest/en/config-options/services/services-extras/_index.md [hunk 2]
```diff
@@ -9,10 +9,14 @@ RKE supports additional service arguments, volume binds and environment variable
...
+
+> **Note:** The default list of admission plugins is defined [here](https://github.com/rancher/kontainer-driver-metadata/blob/master/rke/k8s_service_options.go#L11). The list is sometimes changed per kubernetes version. The exact list per version is defined in the same file.
```
Comment (deniseschannon) [inline content/rke/latest/en/config-options/services/services-extras/_index.md]: can we distinguish between the versions? (RKE v0.3.0+) versus prior to RKE v0.3.0? The file locations are different between the two. One comes from kontainer-driver-metadata and the other comes from types.

File: content/rke/latest/en/config-options/_index.md [hunk 3]
```diff
@@ -67,6 +67,8 @@ kubernetes_version: "v1.11.6-rancher1-1"
 
 In case both `kubernetes_version` and [system images]({{< baseurl >}}/rke/latest/en/config-options/system-images/) are defined, the system images configuration will take precedence over `kubernetes_version`.
 
+> **Note:** In RKE, `kubernetes_version` is used to map services default parameters and options. A detailed list of the options can be found [here](https://github.com/rancher/kontainer-driver-metadata/blob/master/rke/k8s_service_options.go)
```
[the remark this answers is not in the document]
Comment (moelsayed) [reply content/rke/latest/en/config-options/_index.md]: Even though the files locations are different, the content is copied over for older versions (pre v0.3.0). So, it made sense to specify a single location for both versions.

**Comment by deniseschannon** [comment]
@catherineluse can you take this PR and start a new pr with the fixes I requested?

**Comment by catherineluse** [comment]
Will do.

**Comment by catherineluse** [comment]
@deniseschannon Are you sure there need to be more changes? Mohamed said it looks like the same location will work for older RKE version because the content is copied over. https://github.com/rancher/docs/pull/1771#discussion_r332248806

**Comment by deniseschannon** [comment]
@catherineluse Yes, the file location was different for RKE when we released v0.3.0 as it switched from being in `types` repo to the `kontainer-driver-metadata` repo.

**Comment by deniseschannon** [comment]
Close in favor of #1999

**Pull request closed without merging**